src/app/payments/helpers.ts

```diff
@@ -305,3 +306,31 @@ export const newCheckTwoFALimits = async ({
     walletVolume,
   })
 }
+
+export const getPriceRatioForLimits = async <
+  S extends WalletCurrency,
+  R extends WalletCurrency,
+>(
+  paymentFlow: PaymentFlow<S, R>,
+) => {
+  const minSatsForPrecision = 5000n
```

Comment:
should be a config??

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1319/commits/e45dda0dec76c2eeedd643830e6fb4872e6b82cc